### PR TITLE
Coveralls and pylint

### PR DESCRIPTION
**Pull request by Sviridy**

### CODE REVIEW

File: .pylintrc [hunk 1]
```diff
@@ -0,0 +1,625 @@
...
+# Return non-zero exit code if any of these messages/categories are detected,
+# even if score is above --fail-under value. Syntax same as enable. Messages
+# specified are enabled, while categories only check already-enabled messages.
+fail-on=
+
+# Specify a score threshold to be exceeded before program exits with error.
+fail-under=10.0
+
+# Files or directories to be skipped. They should be base names, not paths.
+ignore=CVS
```
Comment (inauros) [inline .pylintrc]: Please add Django system files, configuration files

File: .pylintrc [hunk 2]
```diff
@@ -0,0 +1,625 @@
...
+# Disable the message, report, category or checker with the given id(s). You
+# can either give multiple identifiers separated by comma (,) or put this
+# option multiple times (only on the command line, not in the configuration
+# file where it should appear only once). You can also use "--disable=all" to
+# disable everything first and then reenable specific checks. For example, if
+# you want to run only the similarities checker, you can use "--disable=all
+# --enable=similarities". If you want to run only the classes checker, but have
+# no Warning level messages displayed, use "--disable=all --enable=classes
+# --disable=W".
+disable=print-statement,
```
Comment (inauros) [inline .pylintrc]: please, make this list empty

File: department/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,32 @@
+appdirs==1.4.4
```
Comment (inauros) [inline department/requirements.txt]: there are too many third-party modules in this file, specify only those you use in the project (at least Django and DRF)

File: readme.md [hunk 4]
```diff
@@ -1,3 +1,5 @@
+[![Coverage Status](https://coveralls.io/repos/github/Sviridy/laba/badge.svg)](https://coveralls.io/github/Sviridy/laba)
```
Comment (inauros) [inline readme.md]: Wrong badge configuration, please fix it.

**Review by inauros — CHANGES_REQUESTED** [review]

File: .pylintrc (hunk 1 above)
Comment (Sviridy) [reply .pylintrc]: fixed

File: .pylintrc (hunk 2 above)
Comment (Sviridy) [reply .pylintrc]: fixed

File: department/requirements.txt (hunk 3 above)
Comment (Sviridy) [reply department/requirements.txt]: fixed

File: readme.md (hunk 4 above)
Comment (Sviridy) [reply readme.md]: fixed

**Pull request merged**